src/Compilers/CSharp/Test/Symbol/Symbols/SymbolErrorTests.cs

```diff
@@ -13563,7 +13602,7 @@ interface I
                 Diagnostic(ErrorCode.WRN_ParameterIsStaticClass, "set").WithArguments("C").WithLocation(7, 19),
                 // (8,7): warning CS8897: 'C': static types cannot be used as parameters
                 //     C this[C c] { get; set; } // 5, 6, 7
-                Diagnostic(ErrorCode.WRN_ParameterIsStaticClass, "this").WithArguments("C").WithLocation(8, 7),
+                Diagnostic(ErrorCode.WRN_ParameterIsStaticClass, "C").WithArguments("C").WithLocation(8, 12),
                 // (8,19): warning CS8898: 'C': static types cannot be used as return types
                 //     C this[C c] { get; set; } // 5, 6, 7
                 Diagnostic(ErrorCode.WRN_ReturnTypeIsStaticClass, "get").WithArguments("C").WithLocation(8, 19),
```

Comment:
>Diagnostic(ErrorCode.WRN_ReturnTypeIsStaticClass, "get").WithArguments("C").WithLocation(8, 19), [](http://example.com/codeflow?start=16&length=96)

It feels like we would want to change location for getter as well.

Reply:
Done as part of a7e81d04fe9470107728afac2f48a6bad9eff8f0